exercises/practice/kindergarten-garden/.meta/example.cpp

```diff
@@ -0,0 +1,18 @@
+#include "kindergarten_garden.h"
+
+#include <map>
+
+namespace kindergarten_garden {
+const std::map<std::string, std::string> flower_lookup{
+    {"G", "grass"}, {"C", "clover"}, {"R", "radishes"}, {"V", "violets"}};
+
+std::array<std::string, 4> plants(std::string diagram, std::string student) {
+    size_t row_length{diagram.find('\n')};
+    int index = 2 * (student.at(0) - 'A');
```

Comment:
I'm pretty sure that the trick is exactly that. Changing the tests is a delicate thing on exercism, I don't think I am the only one seeing and using this connection. So there would be some big discussions. If incoming changes would break this "trick" we could re-implement the example solution with the new tests.
/

Reply:
I didn't mean it in the sense that the tests would _actually_ be changed, but more in the sense of writing code that is flexible and extendable. But using this trick is also fine. Just wanted to give you my perspective.